src/layouts/partial-cards.html

```diff
@@ -350,9 +357,13 @@
           states[entityId] = state;
         }
       }
+
+      this.sortGroups = true;
```

Comment:
Compute functions should be pure and not have side effects. Please create a new compute function for sortGroups.